cbclient/api_client.go

```diff
@@ -805,16 +749,48 @@ func (c *CloudBoltClient) authWrappedRequest(req *http.Request) (*http.Response,
 	return resp, nil
 }
 
-// makeRequest wrapps most HTTP requests by adding:
+// makeRequest wraps most HTTP requests by adding:
```

Comment:
I think this comment is out of date - it no longer wraps a request, but creates it with the specified method, url, and body. So this comment could use an update.